src/components/NavBar/LogIn/LogIn.test.js

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+'use strict';
+
+describe('Pursuance login', function() {
+    it('should have a Log in link', function() {
+        browser.url('http://localhost:8080/');
+        expect(browser.getTitle()).to.equal('Pursuance');
+
+        let loginLink = browser.$('=Log In');
+        loginLink.click();
+        expect(true === browser.isVisible('h4=Log In'));
```

Comment:
@kurtwall `h4-Login In`; interesting syntax. What syntax is this?

Reply:
A little sugar that WebdriverIO lays over CSS selectors. It says look for an `h4` tag with the text `Log In`.